py_redact/xlsx_redactor.py

```diff
@@ -0,0 +1,41 @@
+import re
+import logging
+import openpyxl
+from openpyxl.utils.cell import get_column_letter
+
+logger = logging.getLogger("docx-redact")
```

Comment:
Remove logging for now. Need a better way to do this. Will deal later, remove logging from all the project.